widgetastic_manageiq/__init__.py

```diff
@@ -988,7 +988,7 @@ def __init__(self, parent, locator=None, item_name=None, logger=None):
 
     def __locator__(self):
         if not self.locator:
-            self.locator = "//textarea[contains(@id, 'method_data')]"
+            self.locator = "//textarea[contains(@id, 'method_data')]/.."
```

Comment:
If we want the parent element of the textarea, I think its much better to explicitly state it, not use this relative xpath reference.

You should be able to specify this like below, assuming the parent is a `div`:
```
self.locator = "//div[//textarea[contains(@id, 'method_data')]]"
```

The selected element will be the parent div - but only if it contains the child textarea.